dhall-csv/tasty/Main.hs

```diff
@@ -53,3 +60,10 @@ csvToDhallGolden = do
         | csvFile <- csvFiles
         , let dhallFile = replaceExtension csvFile ".dhall"
         ]
+
+encodeCsvDefault :: [Data.Csv.NamedRecord] -> Text
+encodeCsvDefault csv = Data.Text.Encoding.decodeUtf8 $ Data.ByteString.Lazy.toStrict $ Data.Csv.encodeByName header csv
```

Comment:
It might be worth upstreaming everything up to the `Data.Csv.encodeByName` into the `dhall-csv` library since the same logic is also used by the `dhall-to-csv` executable

Reply:
I was unsure of where to put that function because it doesn't really touch anything from Dhall. I think I can create a new `Dhall.Csv.Util` with those functions to encode and decode Csv records